Cargo.toml

```
@@ -5,11 +5,14 @@ name = "h2"
 #   - html_root_url.
 # - Update CHANGELOG.md.
 # - Create git tag
-version = "0.2.0-alpha.3"
+version = "0.2.0"
 license = "MIT"
-authors = ["Carl Lerche <me@carllerche.com>"]
+authors = [
+  "Carl Lerche <me@carllerche.com>",
+  "Sean McArthur <sean@seanmonstar.com>",
+]
 description = "An HTTP/2.0 client and server"
-documentation = "https://docs.rs/h2/0.2.0-alpha.3/h2/"
+documentation = "https://docs.rs/h2/0.2.0/h2/"
```

```suggestion
documentation = "https://docs.rs/h2/0.2.0"
```
I don't think this needs that at the end?

---

It's not needed, but without it redirects to with it, so 🤷‍♂ .